specification/dsdl/dsdl.tex

```diff
@@ -376,6 +431,108 @@ \section{Primitive data types}\label{sec:dsdl_primitive_data_types}
                             \\
 \end{UAVCANSimpleTable}
 
+\subsection{Primitive constant data types}\label{sec:dsdl_primitive_constant_data_types}
+
+In addition to the ordinary data types that are used for defining types to be transmitted over the wire,
+there also exists data types only used for constant evaluation.
+The const data types are used for things that can be calculated only with the information self contained in the dsdl,
+such as defining array lengths, assertions and constant definitions.
+All the primitive const types are given below together with the literal used for construction.
+
+\begin{UAVCANSimpleTable}{Primitive constant data types}{|l l l|}
```

Comment:
I urge you to make the regex patterns formatted using `\verb||`, otherwise they just don't look right. Also, the table doesn't fit on the page (too wide); perhaps a list or some other form should be preferred?

Reply:
I can make another attempt, I generally had a lot of problems with `\verb` inside `UavcanSimpleTable` and started to avoid `\verb` outside normal text.

Reply:
I believe `verb` can be used with any separator, not necessarily `|`. How about `\verb~`?

Reply:
That worked